lib/fusion.flow

```diff
@@ -1839,6 +1843,33 @@ fmerge(a : [Transform<?>]) -> Transform<[?]> {
 		ConstBehaviour([])
 }
 
+fmergeBalanced(a : [Transform<?>]) -> Transform<[?]> {
```

Comment:
I guess we do not need this one after all.

Reply:
removed